app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -1,10 +1,22 @@
+/*global Mousetrap:true*/
 import Controller from "@ember/controller";
 import { Promise } from "rsvp";
 import ModalFunctionality from "discourse/mixins/modal-functionality";
 import discourseComputed from "discourse-common/utils/decorators";
 import { popupAjaxError } from "discourse/lib/ajax-error";
 import { ajax } from "discourse/lib/ajax";
-
+import KeyboardShortcuts from "discourse/lib/keyboard-shortcuts";
+
+// global shortcuts that interfere with these
```

Comment:
Nitpick: why not use longer lines for the comments here? Editors should be at least 80 chars wide.

Reply:
I need to put rules on for vim!